app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
```

Comment:
- [ ] Hello Team, Please notice thou your app is registering a user there is an error `NameError in Devise::RegistrationsController#create` which rises immediately as a user sign up. Please address this for better user experience. 

https://user-images.githubusercontent.com/69077061/223696268-8e0aafa1-79af-4b2d-bfa1-f723ec5d1de6.mp4